### PR TITLE
👕 Set up `Standard JS` linter

### PR DESCRIPTION
**Pull request by jasonrudolph**
Configure linter for [JavaScript Standard Style](https://standardjs.com) to foster consistency with the [Atom style guide](https://github.com/atom/atom/blob/acdf3b20bd94d58cca74e167108278266b2e3670/CONTRIBUTING.md#javascript-styleguide).

### TODO

- [x] Set up [JavaScript Standard Style](https://standardjs.com) linter
- [x] Resolve existing linter violations

### CODE REVIEW

File: lib/toggle-markdown-task.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ function toggleSelection (selection) {
 }
 
 function toggleTask (taskText) {
-  const regex = new RegExp(/([\-\*]\ )(\[[\ x]\])/)
```
Comment (RichardLitt) [inline lib/toggle-markdown-task.js:43]: Is this necessary? Doesn't seem related.

**Comment by RichardLitt** [comment]
Added one comment. Otherwise, LGTM.

File: lib/toggle-markdown-task.js (hunk 1 above)
Comment (jasonrudolph) [reply lib/toggle-markdown-task.js:43]: @RichardLitt: Thanks for the review. :bow:

This change is indeed related, but I should have made that clear in the pull request description. This change resolves existing linter violations related to unnecessary escape characters. See 1b980196a6fea73ffd69150d82456b64916f0e05.

I've updated the pull request description to clarify that this pull request sets up the linter _and_ resolves existing linter violations.
